--- DBD-InterBase-0.41/Makefile.PL.orig	2003-05-20 12:44:30.000000000 +0200
+++ DBD-InterBase-0.41/Makefile.PL	2003-10-31 23:44:19.829509104 +0100
@@ -263,22 +263,6 @@
 # See lib/ExtUtils/MakeMaker.pm for details of how to influence
 # the contents of the Makefile that is written.
 
-# prompt for InterBase bin directory
-$IB_Bin_path = dir_choice("InterBase bin directory", [@ib_bin_dirs], [qw(gfix gfix.exe)]);
-
-unless(-x $IB_Bin_path)
-{
-    carp "I cannot find your InterBase installation.\nDBD::InterBase cannot build or run without InterBase.\n";
-    exit 1;
-}
-
-# get InterBase version
-my $IBVERSION;
-my $GFIX_PATH = $IB_Bin_path . "/" . test_files($IB_Bin_path, [qw(gfix gfix.exe)]);
-chop($IBVERSION = `$GFIX_PATH -z 2>&1`);
-$IBVERSION =~ s/^gfix version //o;
-my $is_final = $IBVERSION =~ /\.6\d+$/ ? 1 : 0;
-
 # prompt for IB include dir
 my $ib_inc = dir_choice("InterBase include directory", [@ib_inc_dirs], [qw(gds.h ibase.h)]);
 
@@ -472,9 +456,6 @@
     exit 1;
 }
 
-# create the test config file
-make_test_conf();
-
 # and last but not least write the makefile
 WriteMakefile(%MakeParams);
 
