diff -Naur DBD-InterBase-0.43.ori/Makefile.PL DBD-InterBase-0.43.mod/Makefile.PL
--- DBD-InterBase-0.43.ori/Makefile.PL	2004-02-24 08:49:01.000000000 +0100
+++ DBD-InterBase-0.43.mod/Makefile.PL	2004-12-08 19:10:46.309183328 +0100
@@ -302,22 +302,6 @@
 # See lib/ExtUtils/MakeMaker.pm for details of how to influence
 # the contents of the Makefile that is written.
 
-# prompt for InterBase bin directory
-$IB_Bin_path = dir_choice("InterBase/Firebird bin directory", [@ib_bin_dirs], [qw(gfix gfix.exe)]);
-
-unless(-x $IB_Bin_path)
-{
-    carp "I cannot find your InterBase/Firebird installation.\nDBD::InterBase cannot build or run without InterBase.\n";
-    exit 1;
-}
-
-# get InterBase version
-my $IBVERSION;
-my $GFIX_PATH = $IB_Bin_path . "/" . test_files($IB_Bin_path, [qw(gfix gfix.exe)]);
-chop($IBVERSION = `$GFIX_PATH -z 2>&1`);
-$IBVERSION =~ s/^gfix version //o;
-my $is_final = $IBVERSION =~ /\.6\d+$/ ? 1 : 0;
-
 # prompt for IB include dir
 my $ib_inc = dir_choice("InterBase/Firebird include directory", [@ib_inc_dirs], [qw(gds.h ibase.h)]);
 
@@ -536,9 +520,6 @@
     exit 1;
 }
 
-# create the test config file
-make_test_conf();
-
 # and last but not least write the makefile
 WriteMakefile(%MakeParams);
 
